pom.xml

```diff
@@ -79,11 +87,79 @@
             <artifactId>plexus-utils</artifactId>
             <version>3.0.22</version>
         </dependency>
+
+        <!-- Windup dependencies -->
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
+            <groupId>org.jboss.windup</groupId>
+            <artifactId>windup-bootstrap</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
You should probably add the rulesets addons here as well, and a comment that these are basically just here to cause maven to predownload them for you.